src/main/java/com/contentgrid/gateway/GatewayApplication.java

```diff
@@ -162,15 +169,16 @@ Customizer<ServerHttpSecurity.AuthorizeExchangeSpec> actuatorsAuthorizeExchangeC
     @Bean
     public SecurityWebFilterChain springWebFilterChain(
             ServerHttpSecurity http,
-            Environment environment,
             ObjectProvider<ReactiveAuthorizationManager<AuthorizationContext>> authorizationManager,
             ServerLogoutSuccessHandler logoutSuccessHandler,
             CorsConfigurationSource corsConfig,
             List<Customizer<OAuth2LoginSpec>> oauth2loginCustomizer,
             List<Customizer<OAuth2ResourceServerSpec>> oauth2resourceServerCustomizer,
             List<Customizer<List<DelegateEntry>>> authenticationEntryPointCustomizer,
             List<Customizer<ServerHttpSecurity.AuthorizeExchangeSpec>> authorizeCustomizer,
-            AuthenticationRefresher authenticationRefresher
+            AuthenticationRefresher authenticationRefresher,
+            Optional<ReactiveAuthenticationManager> reactiveAuthenticationManager,
+            Optional<ReactiveUserDetailsService> reactiveUserDetailsServiceOptional
```

Comment:
Please use consistent names for optionals

Reply:
good spot